src/local_scheduler/local_scheduler_algorithm.cc

```diff
@@ -291,6 +353,8 @@ void handle_actor_worker_connect(LocalSchedulerState *state,
      * filled out, so fill out the correct worker field now. */
     entry->worker = worker;
   }
+  /* TODO(rkn): Is this necessary? */
```

Comment:
It seems like it is, if the actor process takes a while to start up for example.

By the way, it'd be really helpful to have some documentation about the timeline for starting an actor in terms of which events can happen before/after others (e.g., can we submit tasks before the actor has registered)?